docs/docs/admin/monitoring/logging/explorer.md

```diff
@@ -0,0 +1,119 @@
+<!--
+ ~ SPDX-FileCopyrightText: Copyright DB InfraGO AG and contributors
+ ~ SPDX-License-Identifier: Apache-2.0
+ -->
+
+# Grafana Explorer
+
+The Grafana Explorer can be used to view the logs of different components, such
+as the CCM backend or user sessions. The logs are stored persistently, so the
```

Comment:
The abbreviation CCM is only used in one other guide (keycloak), users might not be familiar with it. Consider either expanding the abbreviation on the first usage or just using the full version.

Reply:
Changed it to use the full version.